app.py

```diff
@@ -1,12 +1,22 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+from forms import AddToDoForm
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=session.get_items())
+
+@app.route('/add', methods=['GET', 'POST'])
+def add():
+    form = AddToDoForm()
+    if form.validate_on_submit():
```

Comment:
I'd usually invert this if statement (as failure to validate is the smaller, "off the main branch" case)